hackathon_site/dashboard/frontend/src/slices/event/teamDetailSlice.ts

```diff
@@ -80,6 +120,26 @@ const teamDetailSlice = createSlice({
             state.isLoading = false;
             state.error = payload?.message ?? "Something went wrong";
         });
+        builder.addCase(updateParticipantIdProvided.pending, (state) => {
+            state.isLoading = true;
+            state.error = null;
+        });
+        builder.addCase(updateParticipantIdProvided.fulfilled, (state, { payload }) => {
+            state.isLoading = false;
+            state.error = null;
+
+            const updateObject = {
+                id: payload.id,
+                changes: {
+                    id_provided: payload.id_provided,
+                },
+            };
+            teamDetailAdapter.updateOne(state, updateObject);
+        });
+        builder.addCase(updateParticipantIdProvided.rejected, (state, { payload }) => {
```

Comment:
based on my previous comments, i think this case is not needed anymore

Reply:
removed rejected case for the new thunk